packages/generated-files-bot/src/generated-files-bot.ts

```diff
@@ -165,6 +166,12 @@ export function handler(app: Application) {
     const owner = context.payload.repository.owner.login;
     const repo = context.payload.repository.name;
     const pullNumber = context.payload.pull_request.number;
+    const prAuthor = context.payload.pull_request.user.login;
+
+    // ignore PRs from a configurable list of authors
+    if (config.ignoreAuthors && config.ignoreAuthors.includes(prAuthor)) {
```

Comment:
`if (config.ignoreAuthors?.includes(prAuthor)) {` if you're into the whole brevity thing

Reply:
Done.